frontend/src/scene/main-view/ClassCard.js

```diff
@@ -294,7 +212,7 @@ const Card = class extends React.Component {
                                         .replace('.', ',') + ' €'}
                                 </PriceTag>
                             )}
-                            {!disabled ? (
+                            {errorDetail.type !== 'reserved' ? (
```

Comment:
The user should be able to see the course details even after its reserved. This kind of blocks it.

Reply:
removed